### DISCUSSION TITLE
Using this project for LED-segment display

### DISCUSSION THREAD

**Discussion by JoackimPennerup** [jomjol/AI-on-the-edge-device#815] [Q&A] [answered]
![2022-05-27 11 44 48](https://user-images.githubusercontent.com/13048857/170676556-6d910784-5383-4d1f-a554-f00e545641a7.jpg)
I would like to track the progress on my washing machine. I'm thinking that the foundations are in the project. However I'm a bit unsure how the training process looks. I'm also not completely sure about how much training data one, need to collect? I would also need a different correction algorithm, since it counts down instead of up. 
@jomjol, where would you start if you were to do this? Should I create the hardware setup and just start collecting images for training?

**Comment by savikko** [comment, marked as answer]
I think there are other - more meaningful - ways to track washing machine process.

If you just want some notification when the program has ended, just put some energy measuring smart plug there and you can see (and estimate) when program has been ended. This method has been successfully implemented on multiple cases.

If you want to have more detailed information, just glue acceleration sensor to your washing machine, you can quite easily say from the G-forces that what state the machine is currently in.

**Reply by JoackimPennerup** [reply]
I would like to get an estimate of how long before the current program is done. But it might be possible to get a good read from the power usage and create a suitable model. Thanks.

**Comment by jomjol** [comment]
For LCD, there is a much easier way than CNN. You only need to detect the intensity somewhere on the 7 segment of the LCD and can get the value with a easy table.

If you want to use this, you need to make images of all 10 digits (2-3 each) under the real illumination and camera conditions and then use them in a training: https://github.com/jomjol/neural-network-digital-counter-readout

Most effort will be the setting up of the training environment, the training itself should be rather fast.
